### PR TITLE
Change extraConfigMaps from array to dict

### PR DESCRIPTION
**Pull request by nuwang**
closes: https://github.com/cvmfs-contrib/cvmfs-csi/issues/64

### CODE REVIEW

File: deployments/helm/cvmfs-csi/values.yaml [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
-      {{- if .Values.cache.alien.enabled }}
-      CVMFS_ALIEN_CACHE=/cvmfs-aliencache
-      # When alien cache is used, CVMFS does not control the size of the cache.
-      CVMFS_QUOTA_LIMIT=-1
-      # Whether repositories should share a cache directory or each have their own.
-      CVMFS_SHARED_CACHE=no
-      {{- end -}}
+  # /etc/cvmfs/default.local
+  cvmfs-csi-default-local:
+    data:
```
Comment (gman0) [inline deployments/helm/cvmfs-csi/values.yaml:10]: I think we can make the dict flatter by removing `data`.

**Comment by gman0** [comment]
Thanks for the PR, @nuwang! Other than the comment regarding `data` this looks good.

File: deployments/helm/cvmfs-csi/values.yaml (hunk 1 above)
Comment (nuwang) [reply deployments/helm/cvmfs-csi/values.yaml]: @gman0 Have flattened it further, please take a look.

**Review by gman0 — APPROVED** [review]

**Comment by gman0** [comment]
Thank you @nuwang!

**Pull request merged**